packages/foreman/foreman/foreman.spec

```diff
@@ -2,6 +2,7 @@
 %global confdir extras/packaging/rpm/sources
 %global foreman_rake %{_sbindir}/%{name}-rake
 %global executor_service_name dynflowd
+%global dynflow_sidekiq_service_name dynflow-sidekiq@
```

Comment:
Given the repetition I'm inclined to add `.service` here but I'm fine either way. I'd probably do a cleanup of all service handling later anyway.

Reply:
I would like to stay consistent with the `dynflowd` service, otherwise I would agree :thinking: